test/projectile-test.el

```diff
@@ -41,6 +41,18 @@
      (let ((default-directory sandbox))
        ,@body)))
 
+(defun projectile-mock-serialization-functions (&rest body)
```

Comment:
I think it'd be best if we killed this helper and just used `spy-on` for the functions that need to be mocked, as that's much easier with buttercup.